### DISCUSSION TITLE
Paused Bluetooth Connection

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#595] [Fixed Issues] [answered]
RPi 3A+, Anker SoundCore 2 via Bluetooth, rAudio-1 i20211022

I have been using this combination for a while and it works really well. Until recently if I turned the SoundCore off then RA would continue 'playing' on the RPi so that when I turned the SoundCore back on again the connection was restored and the original audio stream could be heard again. 

With a recent update, I am not sure which, this changed so that when the SoundCore is turned off then the player is stopped. This means that when I turn the SoundCore on again I have to visit the RA web page or use the mobile app to start the audio playing again.  I don't know if this is by accident or design.

The SoundCore has a Play/Pause button, if I pair the SoundCore with my phone then I can use this button to pause or play the existing stream.  This doesn't seem to work with RA - could it ? If this could be done then it might resolve the issue I am having above.

Many thanks

Chris

**Comment by phreatic62** [comment]
I have a similar combination, but a Rpi 3B+. 
If I turn off the speaker, the player pauses, when I turn it back on, I have to wait a few (4-5) seconds after the speaker plays is re-paired tone, but then I can just hit play (Middle of the cover art) and it starts.
The first time I tried the play/pause button on the speaker after reading your comment, it nearly worked, it paused, but did not start, but now it does nothing.

Not wanting to hijack (but they could all be related) but also with this combination, if I use make a clean build from i20211022, the speaker (and my Anker Q30 headphones) pair, but they will not play; the player is working and if I plug into the 3.5mm jack, all is good but no sound will come of of the BT devices. They also show up as the chipset  and A2DP as the description (I can't remember the exact details, but it seems that's a known bug). 
I have a backup from the first time I loaded rAudio about a month ago and that I've run update from settings and pacman -Syu on (pacman was run today) that seems ok.
The only clue was that some files in /etc reported as 775 on the existing install, but as 755 on the new files.

**Comment by ceejayemm** [comment]
Further to my original posting, it seems that if I am playing an album then (sometimes) it does pause when I hit the Play/Pause button on the speaker. Could this be because there is the full set of controls (Play / Pause/ Back/ Forwards/ Stop) when playing albums ?  My usage however for this particular installation is to mainly play webradio stations which when playing only have Play / Stop controls. Thus there is no Pause control available in this scenario.  There must have been a change to some recent version because it used to definitely continue playing web radio stations on the RPI but with no sound output (obviously) rather than the stop / pause that it does with the current version.

At least its not just me.

Thanks

Chris

**Comment by phreatic62** [comment]
I just tried going to the 'Update' path as suggested by rern in your .json thread (#571) and that introduced the BT problem I was describing above :)

**Comment by subatomicglue** [comment]
I have a similar problem with `rAudio-1-64bit-20211022.img`:
- while playing audio (hearing it come out the RPI 4b)
- then changing my audio output to bluetooth, audio continues to play out the RPI
- never have heard the audio out the bluetooth.   
- my jambox says connected;  rAudio-1 has a "green dot" on the bluetooth and bluetooth is listed as the Output
- Reboot doesn't fix.  audio still comes out the pi, even though it appears to connect to bluetooth again
https://github.com/rern/rAudio-1/discussions/600#discussioncomment-1616632

**Reply by subatomicglue** [reply]
My problem was fixed for me by updating to 20211112...   (my problem was total lack of Bluetooth audio which may be different than the OP here)

**Comment by ceejayemm** [comment]
@rern 

I am afraid that update 20211114 does not fix this problem.

Chris

**Comment by ceejayemm** [comment]
Is it possible to fix this please ?  You are obviously detecting when the BT speaker disconnects to stop playback. You also detect when the BT Speaker reconnects (as the BT icon is displayed on the Playback page). So, when the BT Speaker is reconnected, could RA be put back into play mode (like it used to not that long ago).

It really is a pain having to open up a PC or mobile app to simply restart playback and means my wife doesn't use this system anymore when it used to be so simple for her.

Many thanks

Chris

**Reply by rern** [reply]
All right. You've made your point with the "wife" factor. This could lead to a great danger.
So next update:
![autoplay](https://user-images.githubusercontent.com/10094236/144815100-cc5ba41f-5b8a-4c2c-af8f-e29dc0eb4c84.jpg)

**Comment by ceejayemm** [comment]
Thank you very much.  This will save me so much grief.  I will look forward to the next release.

Chris

**Comment by ceejayemm** [comment, marked as answer]
@rern 

Thanks very much for this new feature - works really well.

Thanks

Chris
